### DISCUSSION TITLE
Persistence between round restarts (dead bots stay dead, completed missions stay completed)

### DISCUSSION THREAD

**Discussion by bananaman6213** [FakeFishGames/Barotrauma#13819] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

While doing a Hunting Grounds mission in multiplayer (unstable at the time but the bug seems to have made it to stable), our Sootman bot died. We continued to kill the abyssal, but were sunk to crush depth and returned to the server lobby. Upon retrying the round, Sootman was missing from the crew list and the Hunting Grounds mission was absent. Players had gained experience from the completed mission as well.

I believe the bug still occurs whether you let everyone die or end the round early

Restarting the round a second time seems to fix it

### Reproduction steps

1. Start a round with a mission completable mid-round (Hunting Grounds in our case, but a swarm or beacon may also work)
2. Let a bot die, and complete a mission
3. Either let the whole crew die or return to the server lobby early
4. When restarting, the bots should still be dead and the missions should be absent

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by bananaman6213** [comment]
May be related to #13738

**Comment by Regalis11** [comment]
Thank you for the report! I created [a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13820), we'll look into it!

**Comment by Regalis11** [comment]
We've now fixed this in our private repo, and are planning on hotfixing it as soon as we can. Sorry for the inconvenience!

**Comment by Regalis11** [comment]
Fixed in the latest hotfix
